src/libraries/System.Linq/tests/ConcatTests.cs

```diff
@@ -133,23 +133,6 @@ public void First_Last_ElementAt(IEnumerable<int> _, IEnumerable<int> actual)
             }
         }
 
-        private static void VerifyEqualsWorker<T>(IEnumerable<T> expected, IEnumerable<T> actual)
-        {
-            // Returns a list of functions that, when applied to enumerable, should return
-            // another one that has equivalent contents.
-            var identityTransforms = IdentityTransforms<T>();
-
-            // We run the transforms N^2 times, by testing all transforms
```

Comment:
I'm puzzled by the purpose this method, it all seems entirely unnecessary. Am I missing something?

Reply:
The purpose of the method I deleted because it served no worthwhile purpose? :)

Reply:
Yeah. I'm legitimately curious about what motivated its inclusion back then.

Reply:
Misguided, I think. The stated intent is to validate that all transformations are equivalent to all other transformations, but that doesn't require N^2 tests, and we also don't need to test the test transformations themselves.

I guess it didn't hurt much when it was fast because there were only a few transformations, but my change causes N to grow from 13 to 1099, which means N^2 grows from 169 to 1,207,801... and that difference is very noticeable :)